src/rules/converters/tests/no-submodule-imports.test.ts

```diff
@@ -0,0 +1,34 @@
+import { convertNoSubmoduleImports } from "../no-submodule-imports";
+
+describe(convertNoSubmoduleImports, () => {
```

Comment:
Going off the TSLint rule docs, there should be a test case for a rule arguments list like `["@angular/platform-browser"]`.